src/Satooshi/Bundle/CoverallsV1Bundle/CoverallsV1Bundle.php

```diff
@@ -1,14 +0,0 @@
-<?php
-
-namespace Satooshi\Bundle\CoverallsV1Bundle;
-
-use Symfony\Component\HttpKernel\Bundle\Bundle;
-
-/**
- * Coveralls version 1 API bundle.
- *
- * @author Kitamura Satoshi <with.no.parachute@gmail.com>
- */
-class CoverallsV1Bundle extends Bundle
```

Comment:
Dropping the entire class as we already have a corresponding class (after rename), see https://github.com/satooshi/php-coveralls/blob/v1.0.1/src/Satooshi/Bundle/CoverallsBundle/CoverallsBundle.php.